requirements.txt

```diff
@@ -13,9 +13,10 @@ jsonpickle~=1.2
 expiringdict==1.1.4
 jsonpath-rw
 flask>=1.0.0
-
 pandas~=1.1.1
 PyYAML~=5.1
 mock~=3.0.5
 dataclasses~=0.6
 argparse~=1.4.0
+
+jsonref~=0.2
```

Comment:
This project looks like it hasn't been touched in a good while, and it hasn't even reached a major release. This could cause issues in the future, so keep an eye on this.